content/600-about/200-prisma-docs/20-style-guide/03-spelling-punctuation-formatting.mdx

```diff
@@ -0,0 +1,280 @@
+---
+title: 'Spelling, punctuation, and formatting'
+metaTitle: 'Prisma docs style guide: spelling, punctuation, and formatting'
+metaDescription: 'This section of the style guide provides guidelines on how to present the information you add.'
+tocDepth: 3
+---
+
+## Avoid contractions
+
+In keeping with our conversational style, some contractions are OK:
+
+- Contractions that include the word "not", such as can't, didn't, and doesn't.
+- Contractions that include the words "is" or "are". For example: it's and you're.
+- However, use contractions sparingly.
+
+For example:
+
+- It'll (use "It will")
+
+## Text emphasis
```

Comment:
When we add a section about writing about the UI, we will want to decide if we want to use bold text to indicate components of the UI. That is standard, but we can do as we feel is best.

Reply:
I think we should go with bold, because it's standard, as you say. I'll add a note to this effect.